k8s/prod/common/divorce/div-cos.yaml

```diff
@@ -41,6 +41,8 @@ spec:
         DOCUMENTATION_SWAGGER_ENABLED: "false"
         FEATURE_RESP_SOLICITOR_DETAILS: "true"
         SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_CRON: "0 0 4 ? * * *"
+        BSP_SERVICES_ALLOWED_TO_UPDATE:
```

Comment:
Hm, maybe 

```
BSP_SERVICES_ALLOWED_TO_UPDATE: ""
```
?

Reply:
that would be empty string, which is not the same as empty value - we won't know before we test :)

Reply:
I'd try with empty value and only change it to empty string if it doesn't work

Reply:
yeah, but we use it this way:

`${BSP_SERVICES_ALLOWED_TO_UPDATE:default_value}`

if BSP_SERVICES_ALLOWED_TO_UPDATE is not set -> default_value. I guess.